examples/taxi-ride/README.md

```diff
@@ -174,3 +176,22 @@ $ taxi-ride-fare-logger-7c64c57885-rmc8n  -n taxi-ride-fare
 ```
 $ kubectl cloudflow undeploy taxi-ride-fare
 ```
+
+### Running inside the Sandbox
+
+For running this application in the local sandbox, follow the instructions in [Running in a local Sandbox](https://cloudflow.io/docs/current/get-started/run-in-sandbox.html) document. Once the streamlets start following the `runLocal` command, you will see something like the following displayed on stdout:
+
+```
+logger [taxiride.logger.FarePerRideLogger]
+processor [taxiride.processor.TaxiRideProcessor]
+taxi-fare [taxiride.ingestor.TaxiFareIngress]
+    - HTTP port [3002]
+taxi-ride [taxiride.ingestor.TaxiRideIngress]
+    - HTTP port [3003]
+```
+
+The port numbers indicate the ports for the two ingresses. In order to feed data to the application, you need to run the 2 bash scripts in folder `test-data`, named `send-data-rides.sh` and `send-data-fares.sh`. 
```

Comment:
run and possibly modify, because of the ports? Or should we just change those scripts, so they take a port argument always?

Reply:
If the port is used on the local machine then user will have to change it. Passing the port as an argument would be a good fix although this scenario shouldnt be that common.

Reply:
yeah .. we tried to keep things simple. Just like Stavros mentioned, we tried to keep the port numbers same in the script and the documentation here so that the user does not have to change anything. Agreed that the script accepting ports as arguments is a good fix but adds to an ops complexity for running the script e.g. what happens if the user supplies the ports in the opposite order. But I was not aware that local running generates a different port number.

So in this case it makes sense to change the script to add the port numbers - right ?

Reply:
yes I would just change the scripts, add an arg for port number.

Reply:
@RayRoestenburg 2 args for the 2 ports here (for the 2 ingresses) ..